### ISSUE TITLE
Improve message for `seq_linter()`

### ISSUE THREAD

**Issue by IndrajeetPatil** [r-lib/lintr#1471]
Btw, shouldn't the message here be different:

``` r
library(lintr)

lint(
  text = "1:nrow(data.frame())",
  linters = seq_linter()
)
#> <text>:1:1: warning: [seq_linter] 1:nrow(...) is likely to be wrong in the empty edge case. Use seq_len() instead.
#> 1:nrow(data.frame())
#> ^~~~~~~~~~~~~~~~~~~~
```

<sup>Created on 2022-07-25 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

Current:

> 1:nrow(...) is likely to be wrong in the empty edge case. Use seq_len() instead.

Expected:

> 1:nrow(...) is likely to be wrong in the empty edge case. Use seq_len(nrow(...)) instead.

Since that's the correct replacement:

```r
1:nrow(data.frame())
#> [1] 1 0

seq_len(nrow(data.frame()))
#> integer(0)
```

_Originally posted by @IndrajeetPatil in https://github.com/r-lib/lintr/issues/1468#issuecomment-1193665047_
